openpype/hosts/substancepainter/api/lib.py

```diff
@@ -643,3 +643,55 @@ def _setup_prompt():
         return
 
     return project_mesh
+
+
+def has_rgb_channel_in_texture_set(texture_set_name, map_identifier):
+    """Function to check whether the texture has RGB channel.
+
+    Args:
+        texture_set_name (str): Name of Texture Set
+        map_identifier (str): Map identifier
+
+    Returns:
+        colorspace_dict: A dictionary which stores the boolean
+            value of textures having RGB channels
+    """
+    texture_stack = (
+        substance_painter.textureset.Stack.from_name(texture_set_name)
+    )
+    # 2D_View is always True as it exports all texture maps
+    colorspace_dict = {"2D_View": True}
+    colorspace_dict["BaseColor"] = texture_stack.get_channel(
+        substance_painter.textureset.ChannelType.BaseColor).is_color()
+    colorspace_dict["Roughness"] = texture_stack.get_channel(
+        substance_painter.textureset.ChannelType.Roughness).is_color()
+    colorspace_dict["Metallic"] = texture_stack.get_channel(
+        substance_painter.textureset.ChannelType.Metallic).is_color()
+    colorspace_dict["Height"] = texture_stack.get_channel(
+        substance_painter.textureset.ChannelType.Height).is_color()
+    colorspace_dict["Normal"] = texture_stack.get_channel(
+        substance_painter.textureset.ChannelType.Normal).is_color()
+    return colorspace_dict.get(map_identifier, False)
```

Comment:
The functions wants to return a single value but always queries all channel types - a bit redundant overhead? How about something like this:

```suggestion
    texture_stack = (
        substance_painter.textureset.Stack.from_name(texture_set_name)
    )
    # 2D_View is always True as it exports all texture maps
    if map_identifier == "2D_View":
        return True
    
    channel_type = getattr(substance_painter.textureset.ChannelType, map_identifier, None)
    if channel_type is None:
        return False
        
    return texture_stack.get_channel(channel_type).is_color()
```